miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultSampleAnalyteService.java

```diff
@@ -151,8 +151,8 @@ public void applyChanges(SampleAnalyte target, SampleAnalyte source) throws IOEx
     target.setRegion(source.getRegion());
     target.setTubeId(source.getTubeId());
     target.setStrStatus(source.getStrStatus());
-    target.setGroupId(source.getGroupId());
-    target.setGroupDescription(source.getGroupDescription());
+    target.getSample().getSampleAdditionalInfo().setGroupId(source.getSample().getSampleAdditionalInfo().getGroupId());
+    target.getSample().getSampleAdditionalInfo().setGroupDescription(source.getSample().getSampleAdditionalInfo().getGroupDescription());
```

Comment:
These two lines should be removed. This happens in DefaultSampleAdditionalInfoService